resources/views/layouts/default/flights/simbrief_form.blade.php

```diff
@@ -1,269 +1,411 @@
+                    <tr>
+                      <td>Cont Fuel:</td>
+                      <td>
+                        <select name="contpct" class="form-control">
+                          <option value="auto">AUTO</option>
+                          <option value="0">0 PCT</option>
+                          <option value="0.02">2 PCT</option>
+                          <option value="0.03">3 PCT</option>
+                          <option value="0.05" selected>5 PCT</option>
+                          <option value="0.1">10 PCT</option>
+                          <option value="0.15">15 PCT</option>
+                          <option value="0.2">20 PCT</option>
+                        </select>
+                      </td>
+                    </tr>
+                    <tr>
+                      <td>Reserve Fuel:</td>
+                      <td>
+                        <select name="resvrule" class="form-control">
+                          <option value="auto">AUTO</option>
+                          <option value="0">0 MIN</option>
+                          <option value="15">15 MIN</option>
+                          <option value="30" selected>30 MIN</option>
+                          <option value="45">45 MIN</option>
+                          <option value="60">60 MIN</option>
+                          <option value="75">75 MIN</option>
+                          <option value="90">90 MIN</option>
+                        </select>
+                      </td>
+                    </tr>
+                    <tr>
+                      <td>SID/STAR Type:</td>
+                      <td>
+                        <select name="find_sidstar" class="form-control">
+                          <option value="C">Conventinoal</option>
+                          <option value="R" selected>RNAV</option>
+                        </select>
+                      </td>
+                    </tr>
+                    <tr>
+                      <td>Plan Stepclimbs:</td>
+                      <td>
+                        <select id="stepclimbs" name="stepclimbs" class="form-control" onchange="DisableFL()">
+                          <option value="0" selected>Disabled</option>
+                          <option value="1">Enabled</option>
+                        </select>
+                      </td>
+                    </tr>
+                    <tr>
+                      <td>ETOPS Planning:</td>
+                      <td>
+                        <select name="etops" class="form-control">
+                          <option value="0" selected>Disabled</option>
+                          <option value="1">Enabled</option>
+                        </select>
+                      </td>
+                    </tr>
+                  </table>
+                </div>
+                <br>
+                <div class="form-container-body">
+                  <h6><i class="fas fa-info-circle"></i>&nbsp;@lang('stisla.briefingoptions')</h6>
```

Comment:
Any skin-specific stuff shouldn't be here

Reply:
Sorry about it, missed it while checking